bindings/rust/s2n-tls/src/testing.rs

```diff
@@ -163,15 +200,7 @@ type LocalDataBuffer = RefCell<VecDeque<u8>>;
 pub struct TestPair {
     pub server: connection::Connection,
     pub client: connection::Connection,
-
-    // Pin: since we are dereferencing this pointer (because it is passed as the send/recv ctx)
-    // we need to ensure that the pointer remains in the same place
-    // Box: A Vec (or VecDeque) may be moved or reallocated, so we need another layer of
-    // indirection to have a stable (pinned) reference
-    /// a data buffer that the server writes to and the client reads from
-    server_tx_stream: Pin<Box<LocalDataBuffer>>,
-    /// a data buffer that the client writes to and the server reads from
-    client_tx_stream: Pin<Box<LocalDataBuffer>>,
+    pub(crate) io: TestPairIO,
```

Comment:
```suggestion
    pub io: TestPairIO,
```
Nit: I'd just mark it `pub`. We do expect that in normal usage anyone using this struct will have to access these internal members. Same nit for `TestPairIO`.